### DISCUSSION TITLE
PhantomCameraHost cannot be added as a child for my Camera2D

### DISCUSSION THREAD

**Discussion by Flamebamboo** [ramokz/phantom-camera#325] [Question / Help Wanted]
I am currently on Godot 4.2.2 MacOS, I downloaded the latest version of Phantom camera from the asset library and I encounter this problem only in my main scene,  when I add "PhantomCameraHost" as a child for my Camera2D node the  "PhantomCameraHost" disappears straight away.... Have I missed a step that I'm suppose to do? I have reinstall Godot and remove the plugin and adding it back but the problem still persist....  Anyone could help would be great THX

**Comment by ramokz** [comment]
If the `PCamHost` node is being removed from the tree when you try to add one, then it's likely because you already have one in the scene.

Can you share any screenshots, videos, or even a [minimal-reproducible-project](https://stackoverflow.com/help/minimal-reproducible-example) file? Hard to help otherwise.

**Comment by lightway82** [comment]
I had the same problem for a 3D camera.

**Comment by ramokz** [comment]
Converted this to a discussion post as this reads more like a help request than an issue report.

**Reply by tomdamo** [reply]
I have the same issue too, even the example scenes can't be run. I also notice that I can't find the phantom camera nodes whenever I try to add a new node. 
I am also using a macbook, I already had the phantom camera plugin, deleted the folder in addons and installed it again via the assetLib. I have added screenshots.

Even upon creating a new empty project and installing phantom camera I can not run the example scenes, giving the same errors.
<img width="1677" alt="afbeelding" src="https://github.com/ramokz/phantom-camera/assets/80176908/eb53d76a-19c5-4847-963a-73177591a1ec">
<img width="1680" alt="afbeelding" src="https://github.com/ramokz/phantom-camera/assets/80176908/95969f2e-71d2-45d0-b6da-572c971a0a2e">

screenshot of the empty project
<img width="1670" alt="afbeelding" src="https://github.com/ramokz/phantom-camera/assets/80176908/a184b826-99c1-4e88-a023-7e30cafcf7db">

**Reply by ramokz** [reply]
Is that related to #323?

Also, dumb question, the addon is enabled in the `Plugins` tab in the `Project Settings`, right?

**Reply by tomdamo** [reply]
LMAO, this was the fix. Thank you and my bad, perhaps others with the same issue also did not enable the plugin after reinstalling. 💀

**Reply by ramokz** [reply]
Tbf, Godot ought to prompt the user or enable it by default.
It's kinda strange that you have to manually jump to an entirely different menu after installing it from the Asset Manager.
